deploy/a8s/manifests/postgresql-operator.yaml

```diff
@@ -38,48 +43,54 @@ spec:
+                  meant for read operations and allows for load-balancing across different PostgreSQL cluster
+                  members. Stale reads are possible when reading from this service. By default this feature
+                  is disabled.
                 type: boolean
               expose:
                 default: Internal
                 description: "Expose determines where (and how) the DSI can be accessed
-                  from. Currently supported values are - \"Internal\": the DSI will
-                  be accessible only from inside the K8s cluster - \"LoadBalancer\":
+                  from.\nCurrently supported values are\n- \"Internal\": the DSI will
+                  be accessible only from inside the K8s cluster\n- \"LoadBalancer\":
                   the DSI will receive dedicated load balancers with reachable IP
-                  addresses \t  that can be used from external locations . This is
-                  only supported on K8s clusters that    support external load  balancers.
-                  This field applies to all the services backed by the DSI, that is,
-                  both the master-only one and the read-only one; the \"LoadBalancer\"
-                  value means that each service will get a dedicated LoadBalancer."
+                  addresses\n\t  that can be used from external locations . This is
+                  only supported on K8s clusters that\n   support external load  balancers.\nThis
+                  field applies to all the services backed by the DSI, that is,\nboth
+                  the master-only one and the read-only one; the \"LoadBalancer\"
+                  value means that\neach service will get a dedicated LoadBalancer."
                 enum:
                 - Internal
                 - LoadBalancer
                 type: string
               extensions:
-                description: Extensions defines a list of PostgreSQL extensions which
-                  should be installed. Installing means that the binaries and libraries
-                  of the defined extensions are moved to the PostgreSQL extension
-                  directory. The extensions are NOT loaded by default (i.e. by using
-                  the PostgreSQL "CREATE EXTENSION" command). Updating the list of
-                  extensions will cause a rolling update of the PostgreSQL instance.
+                description: |-
+                  Extensions defines a list of PostgreSQL extensions which should be installed. Extensions can
+                  be installed from the supported list of extensions. NOTE: This implementation of extensions
+                  will enabled in a up coming feature PR. Remove note once PR is merged.
                 items:
                   type: string
+                maxItems: 9
                 type: array
+                x-kubernetes-list-type: set
+                x-kubernetes-validations:
```

Comment:
This validation was merged into [postgresql-opreator](https://github.com/anynines/postgresql-operator/commit/8f4a7a914c4dab0d8ccc27b8ee7aa152f71e7f08) for the new extensions functionality that is not yet included in the main branch. This validation does have a backing implementation. But we are not worse off then we were before.